benchmarks/WebJobs.Script.Benchmarks/CSharpCompilationBenchmarks.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Attributes;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Microsoft.Azure.WebJobs.Script.Extensibility;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Scripting;
+using Microsoft.CodeAnalysis.Scripting;
+using Microsoft.CodeAnalysis.Scripting.Hosting;
+using Microsoft.Extensions.Logging.Abstractions;
+using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Benchmarks
+{
+    public class CSharpCompilationBenchmarks
+    {
+        // Set of samples to benchmark
+        // TODOO: BlobTrigger, needs assembly refs working
+        [Params("DocumentDB", "HttpTrigger", "HttpTrigger-CustomRoute", "HttpTrigger-Weather", "NotificationHub")]
+        public string BenchmarkTrigger;
+
+        // Script source
+        private string ScriptPath;
+        private string GetCSharpSamplePath([CallerFilePath] string thisFilePath = null) =>
+            Path.Combine(thisFilePath, "../../../sample/CSharp");
```

Comment:
Can we replace the `/` with `"..", "..", ".."` so it's also mac friendly :)

Reply:
oo good catch - yes!